themes/custom/lex/scss/molecules/_contact_card.scss

```diff
@@ -0,0 +1,97 @@
+.contact-card {
+    width: 265px;
+    background-color: #EFEFEF;
+    text-align: center;
+    display: inline-block;
+
+
+    .contact-img-container {
+        width: 200px;
+        height: 200px;
+        border-radius: 50%;
+        margin: 20px auto 10px;
+        overflow: hidden;
+        display: inline-block;
+    }
+
+    .contact-img {
+        max-width: none;
+        left: -40px;
+        position: relative;
+        display: inline-block;
+    }
+
+    .contact-heading {
+        color: #0558A7;
+        font-size: 18pt;
+        font-family: 'Open Sans';
+    }
+
+    .contact-subheading {
+        color: #0558A7;
+        font-size: 16pt;
+        font-family: 'Roboto';
+    }
+
+    .description {
+        color: black;
+        font-size: 12pt;
+        font-family: 'Roboto';
+        margin-bottom: 15px;
+    }
+
+    .contact-footer {
+        height: 35px;
+        background-color: #0558A7;
+        width: 100%;
+        margin-top: 10px;
+    }
+
+    .footer-text {
+        a {
+            color: #ffff;
+            font-size: 16pt;
+            font-family: 'Roboto';
+
+            &:hover {
+                text-decoration: none !important;
+                border-bottom: none !important;
+            }
+        }
+    }
+
+    &:hover {
+        width: 291px;
+        background-color: #0558A7;
+
+        .contact-heading {
+            color: white;
+            font-size: 22pt;
```

Comment:
rems if we have to. Stay away from pt/px. Screen sizes are so different.

Reply:
Switched all pt to rems where necessary.